sched/scheduler/jobRunner.go

```diff
@@ -62,7 +61,7 @@ func runJob(job sched.Job, saga *s.Saga, nodes []cm.Node) {
 
 // Logic to execute a single task in a Job.  Ensures Messages are logged to SagaLog
 // Logs StartTask, executes Tasks, Logs EndTask
-func runTask(saga *s.Saga, node cm.Node, taskId string, task sched.TaskDefinition) {
+func runTask(saga *s.Saga, worker worker.WorkerController, taskId string, task sched.TaskDefinition) {
```

Comment:
SendMessage was very specific, because it took a sched.TaskDefinition.

That meant that cluster.Node conflated:
*) discovery
*) communication
*) type

So to implement the Node, you had to implement Worker, so my goal was to decouple that and have Node represent any Node, and then Worker adds to Node by letting you send the relevant Worker messages.